javascript-functions-practice/get-first-element.js

```diff
@@ -0,0 +1,5 @@
+/* exported getFirstElement */
+function getFirstElement(array) {
+  var FirstElement = array.shift();
```

Comment:
This will remove the first item of the array permanently, which is bad news 💀. You should instead retrieve the value without changing the original array